src/qt/stattext.cpp

```diff
@@ -55,6 +55,34 @@ bool wxStaticText::Create(wxWindow *parent,
     m_qtLabel->setBuddy( m_qtLabel );
     m_qtLabel->setTextInteractionFlags( Qt::NoTextInteraction );
 
+    Qt::AlignmentFlag qt_alignment = Qt::AlignAbsolute;
```

Comment:
```suggestion
    Qt::AlignmentFlag qtAlignment = Qt::AlignAbsolute;
```
to conform to the usual naming conventions (alternatively, `alignQt` -- but something in camelCase)